tsconfig.json

```diff
@@ -18,7 +18,7 @@
     "noUnusedLocals": false,
     "baseUrl": "./",
     "skipLibCheck": true,
-    "allowSyntheticDefaultImports": true
+    "allowSyntheticDefaultImports": false
```

Comment:
I think I see why, but remind me to ask you about it tomorrow to confirm

Reply:
why was this working for me...

Reply:
This option doesn't work with ts-node :(

Reply:
It works with jest though, so the tests worked

Reply:
ahhh, interesting